starter_code/views/movie.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="layout-item">
+    <div>
+        <img src="{{movie.image}}" alt="">
+    </div>
+    <div>
+        <a href="/movies">go back</a>
+        <h1>{{movie.title}}</h1>
+        <h2>{{movie.director}}</h2>
+        <h3>Stars</h3>
+
+        <ul>
+            {{#each movie.stars}}
+            <li>{{this}}</li>
+            {{/each}}
+        </ul>
+        <p>
+            {{movie.description}}
+        </p>
+        <div>
+            {{#each movie.showtimes}}
+            <span>{{this}} | </span>
```

Comment:
small tips, instead of adding a  | , you can also just create a border right on your span ;)